config_management/roles/kubernetes-start/tasks/main.yml

```diff
@@ -11,19 +11,19 @@
     enabled: yes
 
 - name: kubeadm init on the master
-  command: 'kubeadm init --token={{ kubernetes_token }}'
```

Comment:
I had to take this out for the moment, something goes a bit weird here and results in an error about invalid toke, which I should investigate.